projects/breviary.json

```diff
@@ -11,8 +11,8 @@
   "ga": "G-Z6B8FX1ZKW",
   "pixel": "1076635099878376",
   "facebook_domain_verification": "o4edch45nwneb17ofcbh98wh7c9miw",
-  "desc_cs": "Breviary je jednoduchá, přehledná a rychlá mobilní a webová aplikace pomáhající zdravotníkům snadno spočítat správné dávkování léků a sledovat jeho historii. Je propojena s potřebnými nemocničními a laboratorními IT systémy a díky tomu umožňuje maximální zjednodušení a automatizaci celého procesu dávkování a administrace léčiv. Způsob výpočtu je plně transparentní a odpovídá nejnovějším vědeckým poznatkům.",
-  "desc_en": "Breviary is an easy to use mobile and web app that helps healthcare workers calculate correct medication dosage and follow the history. The app is fully connected with all relevant hospital IT systems to maximally automatize the process of drug dosage calculation and administration. All dosage calculation methods are fully transparent and follow the latest research.",
+  "desc_cs": "Breviary je jednoduchá, přehledná a rychlá mobilní a webová aplikace pomáhající zdravotníkům snadno spočítat správné dávkování léků a sledovat jeho historii. Aktuálně doporučuje dávkování inzulinu pro diabetes I. typu a heparinu při standardní heparinové terapii. Je propojena s potřebnými nemocničními a laboratorními IT systémy a díky tomu umožňuje maximální zjednodušení a automatizaci celého procesu dávkování a administrace léčiv. Způsob výpočtu doporučení je plně transparentní, byl vyvinut v úzké spolupráci s IKEM a plně odpovídá metodám tam používaným.",
```

Comment:
ja jsem podle me ten heparin a inzulin smazal, nedával bych to sem

Reply:
hmm v tamtom to tam zas pribylo chjo.

Reply:
deleted. TAk na to mrkni a approvni